embedded-systems/README.md

```diff
@@ -0,0 +1,43 @@
+# Embedded-Systems
+
+## Resources
+
+### Beginner Level
+
+* [C Programming For Microcontrollers Featuring ATMEL's Butterfly and the Free WinAVR Complier](http://dsp-book.narod.ru/CPMicro.pdf)
+**_Why this book?:_**
+This book starts from the absolute basics. It presents C programming with an assumption that you are completely new to the programming scene, while simultenously ensuring a healthy level to technical recourse. It is a quick and easy read, with a lot of code and project examples. **_What to keep in min?:_**
+Both the butterfly board and the WinAVR compilers are outdated and suitable substitutes might be required to follow the book properly.
+**_Estimated Time to complete:_**
+20 hours
+
+* [Tiva Launchpad Workshop](https://processors.wiki.ti.com/index.php/Getting_Started_with_the_TIVA%e2%84%a2_C-Series_TM4C123G_LaunchPad?DCMP=tivac&HQS=TM4C123G-Launchpad-Workshop)
+**_Why this works?:_**
```

Comment:
Why this workshop? typo

Reply:
Its not a typo. Why this workshop as in "Why is this workshop useful?"